kano_updater/scenarios.py

```diff
@@ -1256,5 +1256,17 @@ def beta_4_3_0_to_beta_4_3_1(self, dummy_progress):
         pass
 
     def beta_4_3_1_to_beta_4_3_2(self, dummy_progress):
+        # Remove non COPPA compliant apps
+
+        # web whatsapp has no debian candidate, remove manually
+        run_cmd_log('rm -f /usr/share/applications/*whatsapp*')
+        run_cmd_log('/usr/share/icons/Kano/66x66/apps/whatsapp.png')
```

Comment:
Methinks that you are missing the actual command here - you're just running the file!

Reply:
Also, is this file just added through the app store or is it owned by a debian package?

Reply:
Ouch! Got catch! fixed missing command.
The file is not owned by any debian package, that's the thing.